guides/common/modules/proc_configuring-installation.adoc

```diff
@@ -20,7 +20,7 @@ ifdef::foreman-el,katello[]
 For example, on Red Hat-based operating systems, use a utility such as `tmux` or `screen`.
```

Comment:
Unrelated, but `tmux` and `screen` are not Red Hat specific. The whole sections needs a look. Also, `screen` is no longer in EL8 so also the Satellite-specific parts need some attention.